core/proto/keytransparency_v1_grpc/keytransparency_v1_admin_grpc.proto

```diff
@@ -0,0 +1,67 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+option go_package = "github.com/google/keytransparency/core/proto/keytransparency_v1_grpc";
+
+// Key Transparency Administration Service
+//
+// The Key Transparency Service API consists of a map of user names to public
+// keys. Each user name also has a history of public keys that have been
+// associated with it.
+package keytransparency.v1.grpc;
+
+import "proto/keytransparency_v1_proto/keytransparency_v1_admin_proto.proto";
```

Comment:
sort order (should be below google/api/...)

Reply:
done